Added a section of Update parent POM page (#7194)
jenkins-infra/jenkins.io: content/doc/developer/tutorial-improve/_install-apache-maven.adoc

////
// This file is only meant to be included as a snippet in other
// documents.  It provides the same text to multiple files so that we can
// make a single change and have it apply to multiple files
////

== Install Apache Maven

Jenkins plugins are built with Apache Maven.

Download Maven from the link:https://maven.apache.org/download.cgi[Apache Maven website].
Make sure to download one of the binary archives (with `bin` in their name).

NOTE: Many Linux distributions provide packages for Maven for an easier install and upgrade experience.
Consult your distribution's documentation for details.
On macOS, the link:https://brew.sh/[Homebrew] package manager offers Maven packages.
Make sure a recent version of Maven 3, ideally 3.8.6 or newer, is provided if you decide to go this route.

> Next you will need to extract Maven, and take note of its location.

Reviewer: ```suggestion
Next, you will need to extract Maven and take note of its location.
```
adjusting punctuation

> When you extract the Maven files, make sure you extract them directly into the target directory (e.g. extract straight into C:\Program Files (x86)\Maven ) - do not extract the files to a different location then copy the files.

Reviewer: ```suggestion
When you extract the Maven files, make sure you extract them directly into the target directory.
For example, extract the files straight into C:\Program Files (x86)\Maven; do not extract the files to a different location and then copy the files.
```
Suggestion to update the text so that ["e.g." is. not used](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg) per the Microsoft styleguide.
